katalog/loki-triple/README.md

```diff
@@ -0,0 +1,67 @@
+# Loki Triple
+
+<!-- <KFD-DOCS> -->
+
+> 🚨 This package is in technical preview and is subject to change.
+
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system inspired by Prometheus.
+It is designed to be very cost effective and easy to operate.
+It does not index the contents of the logs, but rather a set of labels for each log stream.
+
+`loki-triple` is a high availability setup of Loki, that sets
+up a 3-node cluster of `loki` for a robust and reliable setup.
+
+
+## Requirements
+
+- Kubernetes >= `1.21.0`
+- Kustomize >= `v3.5.3`
+- [prometheus-operator from KFD monitoring module][prometheus-operator]
+- [grafana from KFD monitoring module][grafana]
+
+> Prometheus Operator is necessary since we configure a `ServiceMonitor` to make
+> some metrics available from `loki` on prometheus
+
+## Image repository and tag
+
+* Loki image: `grafana/loki`
+* Loki repo: [Loki on Github][loki-gh]
+
+## Configuration
+
+Fury distribution Loki Stack Single is deployed with the following configuration:
+
+- Single node
```

Comment:
```suggestion
- Three nodes in High Availability mode
```
maybe we can say something about the HA mode, for example, is it active-active? active-passive?

Reply:
I think the best course of action is to remove loki triple in this technical preview, since the boltdb storage cannot be replicated